tools/firtool/firtool.cpp

```diff
@@ -612,6 +612,8 @@ processBuffer(MLIRContext &context, TimingScope &ts, llvm::SourceMgr &sourceMgr,
     return success();
   }
 
+  pm.nest<firrtl::CircuitOp>().addPass(firrtl::createLowerIntrinsicsPass());
```

Comment:
Is it worth running this for `--parse-only` like lower annotations? Might be handy if you're just testing new intrinsics, but it isn't exactly "only parsing" at that point.